.env.example

```diff
@@ -16,6 +16,12 @@ DB_PASSWORD=password
 # The database hostname, e.g. `localhost` for local development or `postgres` for Docker.
 DB_HOST=postgres
 
+# Test database credentials
+TEST_DB_NAME=starttls_test
+TEST_DB_USERNAME=postgres
+TEST_DB_PASSWORD=password
+TEST_DB_HOST=postgres_test
```

Comment:
Do we need to separate out the `TEST_` environment variables? Could we use a separate `.env.test` file which provides a different set of credentials from the default?

Reply:
Can we remove these now?